staff/adri-campo/precourse/airlines/airlines-pro.js

```diff
@@ -0,0 +1,156 @@
+// NORMAL VERSION
+
+// 1. Welcome to the page + ask name. Also, information about all the flights uploaded.
+// 2. Friendly information to the costumer (from/to/price/connection).
+// 3. Average cost of today's flights.
+// 4. List of flights that have connections.
+// 5. Last 5 flights final destination.
+// 6. Final function that executes the full program.
+
+
+// 1.
+
+function welcome() {
+    let userName = "";
+    
+    userName = prompt("Hi, welcome to Skylab Airlines! What's your name?");
+    
+    userName ?  console.log(`Hello, ${userName}!`) : 
+                console.log("Hello!");
+
+    }
```

Comment:
Really, really, really useful!! Many thanks for your comment and for the extension, for sure I will use it from now on, I am pretty sure it will help me a lot!!! 👍🎉